src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -2015,6 +2015,10 @@ private function adaptResourceLocator(
      */
     private function deleteRecursively(NodeInterface $node, $webspace, $dereference = false)
     {
+        foreach ($node->getNodes() as $childNode) {
+            $this->deleteRecursively($childNode, $webspace, $dereference);
```

Comment:
yeah because down all content nods (childs) will be deleted, so all history nodes urls of the child should be deleted to!